packages/web/src/common/store/pages/profile/lineups/feed/sagas.js

```diff
@@ -12,10 +12,9 @@ import {
   waitForAccount,
   makeUid
 } from '@audius/common'
-import { select, call, takeEvery, put } from 'redux-saga/effects'
-
 import { getConfirmCalls } from 'common/store/confirmer/selectors'
 import { LineupSagas } from 'common/store/lineup/sagas'
+import { select, call, takeEvery, put } from 'typed-redux-saga'
```

Comment:
nit: Maybe stick with the `redux-saga/effects` unless we're moving this file to typescript? Alternatively, move this file to typescript? :)